system_tests/seq_filter_test.go

```diff
@@ -0,0 +1,112 @@
+package arbtest
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/arbitrum_types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/params"
+
+	"github.com/offchainlabs/nitro/arbos"
+	"github.com/offchainlabs/nitro/arbos/arbosState"
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbos/l1pricing"
+	"github.com/offchainlabs/nitro/util/arbmath"
+)
+
+func TestSequencerTxFilter(t *testing.T) {
+	t.Parallel()
+
+	builder, header, txes, hooks, cleanup := setupSequencerFilterTest(t, false)
+	defer cleanup()
+
+	block, err := builder.L2.ExecNode.ExecEngine.SequenceTransactions(header, txes, hooks)
```

Comment:
I want to have a block with one tx filtered and one not. e.g. filter transactions with non zero-length data and have one of each

Reply:
Good idea, will add this